### PR TITLE
Add MODELSTORE and MODELEXECUTE

### PR DESCRIPTION
**Pull request by alonre24**
Add the support in the new commands: AI.MODELEXECUTE (AI.MODELRUN is now deprecated) and AI.MODELSTORE (AI.MODELSET is now deprecated).

### CODE REVIEW

File: redisai/command_builder.py [hunk 1]
```diff
@@ -56,7 +124,32 @@ def modeldel(name: AnyStr) -> Sequence:
     return "AI.MODELDEL", name
 
 
-def modelrun(name: AnyStr, inputs: List[AnyStr], outputs: List[AnyStr]) -> Sequence:
+def modelexecute(
+    name: AnyStr,
+    inputs: Union[AnyStr, List[AnyStr]],
+    outputs: Union[AnyStr, List[AnyStr]],
+    timeout: int,
```
Comment (AvitalFineRedis) [inline redisai/command_builder.py:131]: Shouldn't the timeout parameter be optional?

File: test-requirements.txt [hunk 2]
```diff
@@ -6,3 +6,4 @@ nose
 codecov
 numpy
 ml2rt
+deprecated
```
Comment (DvirDukhan) [inline test-requirements.txt:9]: you need to add it to `setup.py` 
also, what is the license of this package?

Comment (alonre24) [reply test-requirements.txt:9]: MIT licence

File: setup.py [hunk 3]
```diff
@@ -6,15 +6,15 @@
 
 setup(
     name="redisai",
-    version="1.0.2",
+    version="1.2.0",
```
Comment (DvirDukhan) [inline setup.py:9]: don't update the version yet on this PR

**Review by DvirDukhan — APPROVED** [review]

**Pull request merged**